edu.stanford.Almond.json

```diff
@@ -0,0 +1,293 @@
+                "--enable-gpl",
+                "--without-compat"
+            ],
+            "build-options" : {
+                "arch" : {
+                    "arm" : {
+                        "config-opts" : [
+                            "--with-build=production",
+                            "--enable-optimized",
+                            "--without-cln",
+                            "--with-gmp",
+                            "--without-readline",
+                            "--enable-gpl",
+                            "--without-compat"
+                        ]
+                    }
+                }
+            },
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://cvc4.cs.stanford.edu/downloads/builds/src/cvc4-1.5.tar.gz",
+                    "sha256" : "5d6b4f8ee8420f85e3f804181341cedf6ea32342c48f355a5be87754152b14e9"
+                }
+            ]
+        },
+        {
+            "name" : "mimic",
+            "config-opts" : [
+                "--enable-shared",
+                "--disable-static",
+                "--with-audio=no"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/MycroftAI/mimic",
+                    "tag" : "1.2.0.2",
+                    "commit" : "67e43bf0fa56008276b878ec3790aa5f32eb2a16"
+                }
+            ]
+        },
+        {
+            "name" : "node",
+            "config-opts" : [
+                "--openssl-use-def-ca-store",
+                "--shared-openssl",
+                "--shared-zlib",
+                "--with-intl=full-icu"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "sha256" : "577c751fdca91c46c60ffd8352e5b465881373bfdde212c17c3a3c1bd2616ee0",
+                    "url" : "https://nodejs.org/dist/v8.11.3/node-v8.11.3.tar.xz"
+                },
+                {
+                    "type" : "archive",
+                    "sha256" : "9542bd97abeaf3ac2f260b6ccaac36eff7dd2d3122e2efe6ca365c3e457489b4",
+                    "url" : "https://ssl.icu-project.org/files/icu4c/60.2/icu4c-60_2-src.zip",
+                    "dest" : "deps/icu"
+                }
+            ]
+        },
+        {
+            "name" : "yarn",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "cp -r . /app/lib/yarn",
+                "ln -s /app/lib/yarn/bin/yarn /app/bin/yarn"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/yarnpkg/yarn/releases/download/v1.7.0/yarn-v1.7.0.tar.gz",
+                    "sha256" : "e7720ee346b2bd7ec32b7e04517643c38648f5022c7981168321ba1636f2dca3"
+                }
+            ]
+        },
+        {
+            "name" : "sqlcipher",
```

Comment:
For aarch64 you need the `rm-configure:true` and:

```
{
					"type": "script",
					"dest-filename": "autogen.sh",
					"commands": [
						"AUTOMAKE=\"automake --foreign\" autoreconf -vfi"
					]
}
```
Like this other example: https://github.com/flathub/io.github.qtox.qTox/blob/209d4096e334d9d29c8689c2ac038177454c5c48/io.github.qtox.qTox.json#L36